ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__multi-nightly.yaml

```diff
@@ -211,6 +211,45 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-byo-kms-etcd-encryption
+- as: aws-ipi-localzone-rootvolume-f7
+  cron: 52 7 3,10,17,24 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      AWS_DEFAULT_GP3_THROUGHPUT: "500"
+      AWS_DEFAULT_MACHINE_VOLUME_SIZE: "120"
+      AWS_DEFAULT_MACHINE_VOLUME_TYPE: gp3
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EDGE_ZONES_LIST: us-east-1-dfw-2a
+      ENABLE_AWS_EDGE_ZONE: "yes"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-edge-zone-rootvolume
+- as: aws-ipi-localzone-rootvolume-f28-destructive
+  cron: 16 9 15 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      AWS_COMPUTE_GP3_THROUGHPUT: "1000"
+      AWS_COMPUTE_VOLUME_SIZE: "120"
+      AWS_COMPUTE_VOLUME_TYPE: gp3
+      AWS_CONTROL_PLANE_GP3_THROUGHPUT: "1200"
+      AWS_CONTROL_PLANE_VOLUME_SIZE: "150"
+      AWS_CONTROL_PLANE_VOLUME_TYPE: gp3
+      AWS_EDGE_GP3_THROUGHPUT: "1000"
+      AWS_EDGE_VOLUME_SIZE: "120"
+      AWS_EDGE_VOLUME_TYPE: gp3
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      EDGE_NODE_WORKER_NUMBER: "1"
+      EDGE_NODE_WORKER_SCHEDULABLE: "no"
+      EDGE_ZONE_TYPES: local-zone
+      EDGE_ZONES_LIST: us-east-1-dfw-2a
+      ENABLE_AWS_EDGE_ZONE: "yes"
+      EXPECTED_COMPUTE_NODE_COUNT: "4"
+      REQUIRED_DEFAULT_STORAGECLASS: gp2-csi
```

Comment:
nit: as `EDGE_NODE_WORKER_NUMBER`, `EDGE_NODE_WORKER_SCHEDULABLE`, `EDGE_ZONE_TYPES`, `EXPECTED_COMPUTE_NODE_COUNT`, `REQUIRED_DEFAULT_STORAGECLASS` are already defined in `cucushift-installer-rehearse-aws-ipi-edge-zone-rootvolume-provision` chain, it's not necessary to put them in job config again, like what you did `- as: aws-ipi-localzone-rootvolume-f7`, another reason is, they are not highly relevant to this job.